core/src/main/java/io/camunda/migrator/config/mybatis/C8Configuration.java

```diff
@@ -388,7 +388,8 @@ public RdbmsService rdbmsService(
         mappingReader,
         batchOperationReader,
         sequenceFlowReader,
-        batchOperationItemReader);
+        batchOperationItemReader,
+        null);
```

Comment:
❓ I see that with the new approach we add beans for all fields. Should we do it for this one too?
Example: https://github.com/camunda/c7-data-migrator/blob/a9ed6b5061cfe273c8c3ec6404dc0cca131d0fc3/core/src/main/java/io/camunda/migrator/config/mybatis/C8Configuration.java#L295-L315